pkg/controllers/managementuser/networkpolicy/netpol.go

```diff
@@ -236,6 +279,42 @@ func (npmgr *netpolMgr) handleHostNetwork(clusterNamespace string) error {
 	return nil
 }
 
+// getClusterCNI returns the cluster's CNI name if it is found or an api error
+func (npmgr *netpolMgr) getClusterCNI(clusterName string) (string, error) {
+	cluster, err := npmgr.mgmtClusters.Get(clusterName, v1.GetOptions{})
```

Comment:
getClusterCNI will be called on each node sync, could we use Listers similar to isNetworkPolicyDisabled check

Reply:
good point, I'll make the change and update